types/numjs/index.d.ts

```diff
@@ -0,0 +1,665 @@
+	 * @returns {NdArray}
+	 */
+	exp(copy?: boolean): NdArray<T>;
+
+	/**
+	 * Calculate the positive square-root of all elements in the array, element-wise.
+	 *
+	 * @param {boolean} [copy=true] - set to false to modify the array rather than create a new one
+	 * @returns {NdArray}
+	 */
+	sqrt(copy?: boolean): NdArray<T>;
+
+	/**
+	 * Return the maximum value of the array
+	 *
+	 * @returns {Number}
+	 */
+	max(): T;
+
+	/**
+	 * Return the minimum value of the array
+	 *
+	 * @returns {Number}
+	 */
+	min(): T;
+
+	/**
+	 * Sum of array elements.
+	 *
+	 * @returns {number}
+	 */
+	sum(): T;
+
+	/**
+	 * Returns the standard deviation, a measure of the spread of a distribution, of the array elements.
+	 *
+	 * @param {object} {ddof:0}
+	 * @returns {number}
+	 */
+	std(): number;
+
+	/**
+	 * Return the arithmetic mean of array elements.
+	 *
+	 * @returns {number}
+	 */
+	mean(): T;
+
+	/**
+	 * Converts {NdArray} to a native JavaScript {Array}
+	 *
+	 * @returns {Array}
+	 */
+	tolist(): T[];
+
+	valueOf(): T[];
+
+	/**
+	 * Stringify the array to make it readable in the console, by a human.
+	 *
+	 * @returns {string}
+	 */
+	inspect(): string;
+
+	/**
+	 * Stringify object to JSON
+	 * @returns {*}
+	 */
+	toJSON(): any;
+
+	/**
+	 * Create a full copy of the array
+	 *
+	 * @returns {NdArray}
+	 */
+	clone(): NdArray<T>;
+
+	/**
+	 * Return true if two arrays have the same shape and elements, false otherwise.
+	 * @param {(Array|NdArray)} array
+	 * @returns {boolean}
+	 */
+	equal<U>(array: njArray<U>): boolean;
+
+	/**
+	 * Round array to the to the nearest integer.
+	 *
+	 * @param {boolean} [copy=true]
+	 * @returns {NdArray}
+	 */
+	round(copy?: boolean): NdArray<T>;
+
+	/**
+	 * Return the inverse of the array, element-wise.
+	 *
+	 * @returns {NdArray}
+	 */
+	negative(): NdArray<T>;
+
+	diag(): NdArray<T>;
+
+	iteraxis(axis: number, cb: (x: NdArray<T>, i: number) => any): void;
+
+	/**
+	 * Returns the discrete, linear convolution of the array using the given filter.
+	 *
+	 * @note: Arrays must have the same dimensions and `filter` must be smaller than the array.
+	 * @note: The convolution product is only given for points where the signals overlap completely. Values outside the signal boundary have no effect. This behaviour is known as the 'valid' mode.
+	 * @note: Use optimized code for 3x3, 3x3x1, 5x5, 5x5x1 filters, FFT otherwise.
+	 *
+	 * @param {Array|NdArray} filter
+	 */
+	convolve(filter: njArray<T>): NdArray<T>;
+
+	fftconvolve(filter: njArray<T>): NdArray<T>;
+}
+
+export type ndArrayData<T> = BaseNdArray.Data<T>;
+export type njArray<T> = ndArrayData<T> | NdArray<T>;
+export type njParam<T> = njArray<T> | number;
+
+/**
+ * Return absolute value of the input array, element-wise.
+ *
+ * @param {Param} x
+ * @returns {NdArray}
+ */
+export function abs<T = number>(x: njParam<T>): NdArray<T>;
+
+/**
+ * Add arguments, element-wise.
+ *
+ * @param {Param} a
```

Comment:
Don't include JSDoc comments that only provide information that's already in the code.